orchestration/templates/export-diff.yaml

```diff
@@ -39,9 +39,11 @@ spec:
           {{- $datePresent := "{{tasks.is-preceding-date-present.outputs.result}} > 0" }}
 
           - name: write-release-date
-            dependencies: [is-preceding-date-present, check-processing-needed]
             {{- if $versionIsPinned }}
+            dependencies: [is-preceding-date-present, check-processing-needed]
```

Comment:
That's annoying; would it be easier/possible to always run check-processing-needed and make it a no-op, instead of doing the check here?